Tests/SynchronizableTests.swift

```diff
@@ -23,12 +23,28 @@ import Foundation
 import XCTest
 
 
+extension Valet {
+
+    // MARK: Accessibility -> CloudAccessibility conversion
+
+    var cloudAccessibility: CloudAccessibility {
```

Comment:
thoughts here?

Reply:
I don't think we need this. All we're doing here is unwrapping the associated value in the `Flavor` enum. Let's just get rid of `Flavor`

`CloudAccessibility(rawValue: accessibility)` will work once we surface a `var accessibility: Accessibility`